mineinabyss-features/src/main/kotlin/com/mineinabyss/misc/MiscFeature.kt

```diff
@@ -2,14 +2,13 @@ package com.mineinabyss.misc
 import com.mineinabyss.idofront.plugin.registerEvents
 import com.mineinabyss.mineinabyss.core.AbyssFeature
 import com.mineinabyss.mineinabyss.core.MineInAbyssPlugin
-import com.mineinabyss.misc.InteractionListener
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 
 @Serializable
 @SerialName("invisibleItemFrames")
-class InvisibleItemFramesFeature: AbyssFeature {
+class MiscFeature: AbyssFeature {
```

Comment:
Change the SerialName aswell to misc

Reply:
[d86b1a6](https://github.com/MineInAbyss/MineInAbyss/pull/158/commits/d86b1a6a31b8a67c17da1cf1a74061c3d8e0e6f3) closes this